components/arrow-left/src/test.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
-import { mount, shallow } from 'enzyme';
+import { shallow } from 'enzyme';
 import { createMatchers } from 'jest-emotion';
 import * as emotion from 'emotion';
 import Component from '.';
```

Comment:
super-minor comment...

i'd prefer to see separation between the library components we're importing, and the local component that we're testing in our test files

i.e. i'd like to see a new line before this import :grin:

Reply:
I agree! :) `Arrow` was the only one that had this. I've also brought all tests into alignment with the `import Component from '.'` convention that I saw used.